### DISCUSSION TITLE
Does the plugin support ItemsAdder items?

### DISCUSSION THREAD

**Discussion by ElMoha943** [TownyAdvanced/TownyResources#78] [Q&A] [answered]
Can a town produce custom items made with [ItemsAdders](https://www.spigotmc.org/resources/%E2%9C%A8itemsadder%E2%AD%90custom-items-armors-hud-gui-mobs-emoji-blocks-wings-hats-liquids.73355/)? If not, could you add support for said plugin?

**Comment by LlmDl** [comment, marked as answer]
Currently ItemsAdder is not supported. I would have to research what is involved in their API, I have not purchased this plugin so I would not be able to test it.

**Reply by ElMoha943** [reply]
I offer to test it if you want, also i link you to their [adding support tutorial](https://itemsadder.devs.beer/compatibility-with-other-plugins/adding-compatibility) aswell as their [discord](https://discord.gg/EDUdPzKgfs)
